styling/src/jsMain/kotlin/dev.fritz2.styling/params/typography.kt

```diff
@@ -162,7 +162,7 @@ interface Typo : StyleParams {
         lg: (Fonts.() -> Property)? = null,
         xl: (Fonts.() -> Property)? = null
     ) =
-        property(fontSizeKey, Theme().fonts, sm, md, lg, xl)
+        property(fontFamilyKey, Theme().fonts, sm, md, lg, xl)
```

Comment:
Please correct the API doc example call too! ``body`` seems not to a an appropriate value there!

Reply:
no correction needed